src/it/java/org/eclipse/microprofile/starter/utils/MPSpecGET.java

```diff
@@ -61,7 +61,10 @@
     //}),
     REST_CLIENT(new String[][]{
             new String[]{"data/client/test/parameterValue=xxx", "Processed parameter value 'parameterValue=xxx'"}
-    });
+    }),
+    GRAPH_QL(new String[][]{
+            new String[]{"data/client/test/parameterValue=xxx", "Wolverine"}
```

Comment:
I don't think this this is the valid url for graphQL though

Reply:
It is not. A stray thing....   this is a draft :-)